command-expansion-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -2,46 +2,122 @@
 export class Command<A extends ArgType[] | { [argName: string]: any } = [] | {}> {
   public readonly stem: string;
   public readonly arguments: A;
+  public readonly absoluteTime?: Temporal.Duration;
```

Comment:
Let's make these `Temporal.Duration | null` so we're explicit about there being _nothing_ in there. undefined is kind of weak in that bad indexing or undefined variables can produce undefined